presto-tpch/src/main/java/com/facebook/presto/tpch/TpchMetadata.java

```diff
@@ -119,6 +127,18 @@
             .map(value -> new NullableValue(getPrestoType(PartColumn.CONTAINER), value))
             .collect(toSet());
 
+    private static final Map<String, List<ColumnMetadata>> EXTRA_TABLES = ImmutableMap.<String, List<ColumnMetadata>>builder()
+                .put(
+                        "lineitem_ext",
```

Comment:
I would put a comment here.  We're going to remove it before master right?

Reply:
@tdcmeehan We'll need something like this to test on master as well. One option would be to add a new TpchPlusConnector and leave TpchConnector being purely TPC-H. Another option would be to allow extended TPC-H connector like this.